### DISCUSSION TITLE
Is there a node or function to make UI auto add Lora trigger words to prompt?

### DISCUSSION THREAD

**Discussion by koopa990** [Comfy-Org/ComfyUI#3524] [Q&A]
I dont see any such options so far.

**Comment by andreszs** [comment]
Not all loras require trigger words. Anyway, how could comfyui possibly retrieve such words from the lora?

**Reply by koopa990** [reply]
I found a custom node that retrieves them.

After spending many hours on comfy--messing around with nodes and how to perfectly replicate my images from WebUI--maybe this is not for me. I was ONLY here for faster processing speed and less resource usage on my system...

**Reply by CasualDev242** [reply]
> I found a custom node that retrieves them.
> 
> After spending many hours on comfy--messing around with nodes and how to perfectly replicate my images from WebUI--maybe this is not for me. I was ONLY here for faster processing speed and less resource usage on my system...

Would you mind sharing the name of the custom node?
